cmd/radar/radar.go

```diff
@@ -196,9 +196,82 @@ func main() {
 		} else {
 			staticHandler = http.FileServer(http.FS(radar.StaticFiles))
```

Comment:
@patrickod I'm not 100% clear on how the embed.FS StaticFiles works. should we be using it for files built by the frontend?

This PR can move ahead, but I'm wondering if there's an improvement/optimisations we should be using